pkg/services/org/org.go

```diff
@@ -2,9 +2,12 @@ package org
 
 import (
 	"context"
+
+	"github.com/grafana/grafana/pkg/registry"
 )
 
 type Service interface {
+	registry.ProvidesUsageStats
```

Comment:
Instead of exposing this function in public interface we should move the code to gather stats into `userimpl.Service` because we collect them from there.

Reply:
Changed as requested